ionic_user_interface/src/views/ViewRoute.vue

```diff
@@ -4,36 +4,52 @@
       <div id="container" class="ion-text-left">
         <ion-img :src="routeDetails.routeURL"></ion-img>
         <ion-row class="ion-justify-content-between">
-          <strong>{{ routeDetails.routeName }}</strong>
+          <div class="route-title">
+            <b>{{ routeDetails.routeName }}</b>
+          </div>
           <VoteButton
+            class="margin-right"
             :username="routeDetails.username"
             :createdAt="routeDetails.createdAt"
             v-model:voteCount="routeDetails.voteCount"
             v-model:hasVoted="routeDetails.hasVoted"
           ></VoteButton>
         </ion-row>
-        <div v-if="isLoggedIn">
-          <ion-button
-            v-if="!hasReported"
-            color="danger"
-            @click="() => reportRouteHandler(routeDetails.username, routeDetails.createdAt)"
+        <ion-row class="ion-justify-content-between display-flex">
+          <ion-item
+            class="ion-no-padding margin-left rounded"
+            :href="'/userRoutes/' + routeDetails.username"
           >
-            <ion-label>Report this route&nbsp;</ion-label>
-            <ion-icon :icon="flag"></ion-icon>
-          </ion-button>
-          <ion-button disabled v-if="hasReported" color="medium">
-            <ion-label>Reported</ion-label>
-          </ion-button>
-        </div>
+            <ion-icon
+              class="margin-right margin-left"
+              slot="start"
+              :icon="personCircleOutline"
+            ></ion-icon>
+            <ion-label class="align-middle">{{ routeDetails.username }}</ion-label>
+          </ion-item>
+          <div v-if="isLoggedIn" class="margin-right">
```

Comment:
I think this one might need margin left too, especially on mobile

Reply:
added but it makes no difference

Reply:
![image](https://user-images.githubusercontent.com/19868916/125904192-f7bee61d-75a7-4b0e-adda-59ab75e7212e.png)

I meant it looks weird

Reply:
oh lol I alr reported everything so I couldn't see this. There should be a way to unreport man